waspc/src/Wasp/Generator/DockerGenerator.hs

```diff
@@ -10,24 +10,29 @@ import StrongPath (File', Path', Rel, relfile)
 import Wasp.AppSpec (AppSpec)
 import qualified Wasp.AppSpec as AS
 import qualified Wasp.AppSpec.Entity as AS.Entity
-import Wasp.Generator.Common (ProjectRootDir)
+import Wasp.Generator.Common (ProjectRootDir, latestMajorNodeVersion)
 import Wasp.Generator.FileDraft (FileDraft, createTemplateFileDraft)
 import Wasp.Generator.Monad (Generator)
+import Wasp.Generator.ServerGenerator (areServerPatchesUsed)
 import Wasp.Generator.Templates (TemplatesDir)
+import qualified Wasp.SemanticVersion as SV
 
 genDockerFiles :: AppSpec -> Generator [FileDraft]
 genDockerFiles spec = sequence [genDockerfile spec, genDockerignore spec]
 
 -- TODO: Inject paths to server and db files/dirs, right now they are hardcoded in the templates.
 genDockerfile :: AppSpec -> Generator FileDraft
-genDockerfile spec =
+genDockerfile spec = do
+  usingServerPatches <- areServerPatchesUsed spec
   return $
     createTemplateFileDraft
       ([relfile|Dockerfile|] :: Path' (Rel ProjectRootDir) File')
       ([relfile|Dockerfile|] :: Path' (Rel TemplatesDir) File')
       ( Just $
           object
-            [ "usingPrisma" .= not (null $ AS.getDecls @AS.Entity.Entity spec)
+            [ "usingPrisma" .= not (null $ AS.getDecls @AS.Entity.Entity spec),
+              "nodeMajorVersion" .= show (SV.major latestMajorNodeVersion),
```

Comment:
How did this become part of this PR, because you learned about this on the way? NO problem with it, just checking that  you did not commit this by accident!

Reply:
It was discovered during the development of this feature and I just decided to fix it during since I was already in this file.